test/env

```diff
@@ -87,6 +101,10 @@ start_topology()
 	ln -sf "$file" "$envdir/qeneth/$filename"
     done
 
+    base_img=$(get_base_img "$files")
+    output_img=$(get_output_img "$base_img")
+    $testdir/inject-test-mode -b "$base_img" -o "$output_img"
```

Comment:
Not sure `get_output_img()` brings that much to the table here, how about short-circuiting it:

```suggestion
    $testdir/inject-test-mode -b "$base_img" -o "${base_img%-disk.img}-disk-test.img"
```

Reply:
agree, thnx :)